bin/bundle

```diff
@@ -57,17 +55,15 @@ m = Module.new do
 
   def lockfile_version
     return unless File.file?(lockfile)
-
     lockfile_contents = File.read(lockfile)
-    return unless lockfile_contents =~ /\n\nBUNDLED WITH\n\s{2,}(#{Gem::Version::VERSION_PATTERN})\n/o
-
+    return unless lockfile_contents =~ /\n\nBUNDLED WITH\n\s{2,}(#{Gem::Version::VERSION_PATTERN})\n/
```

Comment:
Performance/ConstantRegexp: Extract this regexp into a constant, memoize it, or append an `/o` option to its options.